pkg/render/render.go

```diff
@@ -109,14 +109,15 @@ func Calico(
 	ns.ObjectMeta = metav1.ObjectMeta{Name: ns.Name, Namespace: common.CalicoNamespace}
 	tss = append(tss, ts, ns)
 
-	if managerTLSSecret == nil && cr.Spec.Variant == operator.TigeraSecureEnterprise && cr.Spec.ClusterManagementType == operator.ClusterManagementTypeManagement {
-		// Generate CA and TLS certificate for tigera-manager
-		log.Info("Creating secret for manager")
+	if managerInternalTLSSecret == nil && cr.Spec.Variant == operator.TigeraSecureEnterprise && cr.Spec.ClusterManagementType == operator.ClusterManagementTypeManagement {
+		// Generate CA and TLS certificate for tigera-manager for internal traffic within the K8s cluster
+		// The certificate will be issued for ManagerServiceDNS and localhost
+		log.Info("Creating secret for internal manager credentials")
 		var err error
-		managerTLSSecret, err = CreateOperatorTLSSecret(nil,
```

Comment:
don't we still need to create this somewhere?

Reply:
@ozdanborne `manager-tls` will either be created by the operator in https://github.com/tigera/operator/pull/606/files#diff-29e68fd54befc00ec93a59c7359836d0R90 or a client is going to provided it.

Reply:
If this section now generates `internal-manager-tls` instead, it doesn't really need the `localhost` domain in the certificate (since it is intended to be called only from components that have access to the service DNS name). 

It probably doesn't make it less secure, but it's just not really needed for `internal-manager-tls`. The `localhost` was useful for `manager-tls` when a user tries to access Manager UI through port-forwarding https://localhost:9443.